### PR TITLE
adding application with old images

### PR DESCRIPTION
**Pull request by armsteadj1**
## Description

- Adding Application Concept page

## Testing required outside of automated testing?

- [ ] Not Applicable

### Screenshots (if appropriate):

- [ ] Not Applicable

## Rollback / Rollforward Procedure

- [ ] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: collections/concepts/applications-concept.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+- Management
+    - Used with scripts or the Basis Theory Terraform (request access here) modules to configure your [Tenant’s](https://guides.basistheory.com/concepts/tenants/) settings (Applications, Reactors, etc) without logging into our UI Portal.
+
+### Permissions
+
+Each Application Type has different levels of permission available for the application. These permissions range from enabling the ability to Create Tokens, Use Tokens with our Proxy or Reactors, all the way to the ability to create new applications. You can find a detailed list of all of our permissions for all of our [Application types here.](https://docs.basistheory.com/api-reference/#permissions-permission-types)
+
+### API Key
+
+Your Application’s API Key is used to authenticate your systems to the Basis Theory platform. You will use this with our API via our `BT-API-KEY` header or into our Elements SDK’s to grant access to collect and secure data from the browser.  These API Keys follow a format similar to `key_4qUtg83907AnDem90aJSzcN`.
```
Comment (koalaty-code) [inline collections/concepts/applications-concept.md:44]:
```suggestion
Your Application’s API Key is used to authenticate your systems to the Basis Theory platform. The API Key can be used to make authenticated requests to our API directly via the `BT-API-KEY`, using Hosted Elements, or using the Basis Theory SDK. These API Keys follow a format similar to `key_4qUtg83907AnDem90aJSzcN`.

```

Trying to capture all the ways an API Key can be used to interact with BT. We could add links to our API Docs and Hosted Elements. Do we have somewhere we could link to cover all the SDKs we offer?

File: collections/concepts/applications-concept.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+A common use-case for Basis Theory is to grant multiple systems access to the same data or a subset of the data that is actively being secured in a [Tenant](https://guides.basistheory.com/concepts/tenants/). This could mean that one system is allowed to only collect the data, one system is allowed to view the data, and the last system is only allowed to [Proxy](https://guides.basistheory.com/concepts/what-is-the-proxy/) the data to another 3rd party but never access the raw data.
+
+<img src="/assets/images/concepts/applications_each_system.png">
+
+### Enable partners to use data secured by Basis Theory
+
+Often businesses need to grant their partners access to their sensitive data — traditionally these businesses would just export the needed data and send it over to their partner. With Basis Theory Applications, a business creates an Application with only permissions restricted to ONLY the data that partner needs. In some cases, businesses will not allow the partner to see the raw data, rather allow them [Proxy](https://guides.basistheory.com/concepts/what-is-the-proxy/) the data to another API or use write serverless code with Reactors.
```
Comment (koalaty-code) [inline collections/concepts/applications-concept.md:58]:
```suggestion
Often, businesses need to grant their partners access to their sensitive data — traditionally, these businesses would just export the needed data and send it over to their partner. With Basis Theory Applications, a business creates an Application with permissions restricted to ONLY the data that partner needs. In some cases, businesses will not allow the partner to see the raw data, but rather allow them to [Proxy](https://guides.basistheory.com/concepts/what-is-the-proxy/) the data to another API or run serverless functions with Reactors.

```

File: collections/concepts/applications-concept.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+<img src="/assets/images/concepts/applications_partners.png">
+
+### Manage Basis Theory assets with Infrastructure as Code
+
+The Management application type allows the Basis Theory platform to be configured entirely via our API. This enables engineers to write scripts to set up new Applications or Manage your Custom Reactor Formulas all without logging into our Portal UI. 
+
+Interested in using our Terraform module? [Reach out!](https://basistheory.com/contact)
```
Comment (koalaty-code) [inline collections/concepts/applications-concept.md:66]:
```suggestion
Interested in using IaC (Infrastructure as Code) tooling such as Terraform to configure Basis Theory? [Reach out!](https://basistheory.com/contact)

```

File: collections/concepts/applications-concept.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+### Name
+
+The name of your Application gives you clarity on what context it is being used for, for example, “Acme Billing App” or “Acme Mobile App”. 
+
+### Application Types
+
+- Server-to-Server
+    - Used with a traditional web application that runs their application logic primarily on a server or in a serverless environment (e.g. NodeJs, .NET, Java, or Ruby). This application type has the most permissions available as the API Key will be private and secured within a server environment.
+- Elements
+    - Used with browser-based applications (e.g. Vanilla Javascript, React, Angular, etc) using the Basis Theory Elements to collect and secure data.
```
[the remark this answers is not in the document]
Comment (armsteadj1) [reply collections/concepts/applications-concept.md:32]:
```suggestion
    - Used with browser-based applications (e.g. Vanilla Javascript, React, Angular, etc) using the Basis Theory Elements to collect and secure data. Applications created with this type have limited permissions to enable using the API Key in publicly available code.
```

File: collections/concepts/applications-concept.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+
+### Permissions
+
+Each Application Type has different levels of permission available for the application. These permissions range from enabling the ability to Create Tokens, Use Tokens with our Proxy or Reactors, all the way to the ability to create new applications. You can find a detailed list of all of our permissions for all of our Application Types [here](https://docs.basistheory.com/api-reference/#permissions-permission-types).
+
+
+### API Key
+
+Your Application’s API Key is used to authenticate your systems to the Basis Theory platform. The API Key can be used to make authenticated requests to our API directly via the `BT-API-KEY`, using Hosted Elements, or using the Basis Theory SDK. These API Keys follow a format similar to `key_4qUtg83907AnDem90aJSzcN`.
```
Comment (koalaty-code) [inline collections/concepts/applications-concept.md:47]:
```suggestion
Your Application’s API Key is used to authenticate your systems to the Basis Theory platform. The API Key can be used to make authenticated requests to our API directly, using Hosted Elements, or using the Basis Theory SDK. These API Keys follow a format similar to `key_4qUtg83907AnDem90aJSzcN`.

```

File: collections/concepts/applications-concept.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+The Management application type allows the Basis Theory platform to be configured entirely via our API. This enables engineers to write scripts to set up new Applications or Manage your Custom Reactor Formulas all without logging into our Portal UI. 
+
+Interested in using IaC (Infrastructure as Code) tooling such as Terraform to configure Basis Theory? [Reach out!](https://basistheory.com/contact)
+
+
+<img src="/assets/images/concepts/applications_iac.jpeg">
+
+---
+
+## Answer any Common Questions:
```
Comment (koalaty-code) [inline collections/concepts/applications-concept.md:79]:
```suggestion
## FAQs

```

**Review by koalaty-code — APPROVED** [review]

**Pull request merged**